qiita_db/analysis.py

```diff
@@ -780,20 +783,32 @@ def build_files(self, merge_duplicated_sample_ids):
             # are going to create
             rename_dup_samples = False
             grouped_samples = {}
-            for k, v in viewitems(samples):
-                a = qdb.artifact.Artifact(k)
-                p = a.processing_parameters
-                if p is not None and p.command is not None:
-                    ref = (str(p.values['reference'])
-                           if 'reference' in p.values else 'na')
-                    cid = str(p.command.id)
+            for aid, asamples in viewitems(samples):
+                # find the artifat info, [0] there should be only 1 info
+                ainfo = [bi for bi in bioms_info
+                         if bi['artifact_id'] == aid][0]
+
+                data_type = ainfo['data_type']
+                algorithm = ainfo['algorithm']
+                target_subfragment = ainfo['target_subfragment']
+                parameters = ['%s: %s' % (k, v)
+                              for k, v in viewitems(ainfo['parameters'])]
+                files = ainfo['files']
+
+                l = "%s || %s || %s || %s" % (
+                    data_type, algorithm, ','.join(target_subfragment),
+                    ', '.join(parameters))
+                # deblur special case, we need to account for file name
+                if 'deblur-workflow' in algorithm:
+                    # [0] there is always just one biom
+                    l += " || %s" % [f for f in files
```

Comment:
In other formatting instances the list is joined (`''.join(some_list)`), any reason why you are not joining this list?